Makefile.orbis

```diff
@@ -78,7 +78,7 @@ else
 	HAVE_THREAD_ATTR			:= 1
 	HAVE_LIBRETRODB				:= 1
 	HAVE_CC_RESAMPLER			:= 1
-	HAVE_CHEEVOS					:= 1
+	HAVE_CHEEVOS					:= 0
```

Comment:
This change fixes the [PS4/ORBIS build](https://github.com/libretro/RetroArch/actions/runs/4974992400/jobs/8901768167?pr=15285#step:5:156).

The Makefile was setting a flag indicating that achievements were supported, causing additional files to be included in the build. However, it didn't add the `-DHAVE_CHEEVOS` flag to the compile flags (line 124 below). Doing so caused [linking errors](https://github.com/libretro/RetroArch/actions/runs/4975158241/jobs/8902061515?pr=15285#step:5:266), so I opted for disabling the flag.

Not having the compile flag (and the resulting linking errors) suggest achievements weren't actually supported, so I believe this change to be correct. Let me know if something else needs to be done.